cmd/network-slirp-binding/slirp.go

```diff
@@ -32,10 +32,16 @@ import (
 
 	"kubevirt.io/client-go/log"
 
+	"kubevirt.io/kubevirt/cmd/network-slirp-binding/dns"
 	srv "kubevirt.io/kubevirt/cmd/network-slirp-binding/server"
 )
 
 func main() {
+	searchDomains, err := dns.ReadResolvConfSearchDomains()
+	if err != nil {
+		panic(err)
```

Comment:
IMO it should exist normally with a message to stderr (using the log) and not panic. Panic fits libraries perhaps, not main().

Reply:
I see you kept the panic.

Reply:
It fixed in the second commit https://github.com/kubevirt/kubevirt/pull/10272/commits/daa84f7f9d801a4afe522b0bf94f1fda1be4e254 
I think Github had a hiccup.